requirements_dev.txt

```diff
@@ -4,6 +4,7 @@ pip==20.2.2
 # base dependencies
 numpy==1.19.1
 scipy==1.5.2
+psutil==5.7.3
```

Comment:
Is adding this dependency fine? [psutil](https://github.com/giampaolo/psutil) is the most reliable way to get the total RAM size in multiple platforms. 6.8k stars in GitHub.